scripts/ci/30_test.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -euxo pipefail
 
+: "${DD_CIVISIBILITY_AGENTLESS_ENABLED:=true}"
+: "${DD_SITE:=datadoghq.eu}"
+: "${DD_ENV:=dev}"
+: "${DD_SERVICE:=unit-test-action}"
+: "${DD_API_KEY:='<set-api-key>'}"
+: "${NODE_OPTIONS:=--max-old-space-size=8192}"
```

Comment:
Is it not redundant to have the max old space config here since it is also defined below?

Reply:
This allows overriding/adding settings using environment variables. Below we are setting some options we think are a minimum. That way we will always have at least `--max-old-space-size=8192`, even if the user sets `NODE_OPTIONS` to e.g. `--verbose` (which would disable `--max-old-space-size`).

Reply:
Though, the default in line 9, is indeed redundant.